### PR TITLE
add cpod level resource stat

### PR DESCRIPTION
**Pull request by congpeiqing**
# Describe your changes
add cpod level resource stat

## Issue
- [ ] Check this box if this PR fixes an issue, and then paste the link below
- [ ] No associated issue

## Test:
- [ ] Check this box if this PR has tests
- [ ] No test needed

### CODE REVIEW

**Review by glen-nascentcoreai — APPROVED** [review]
LGTM. Let's refine later

File: manager/pkg/resource/resource.go [hunk 1]
```diff
@@ -67,11 +68,29 @@ type NodeInfo struct {
...
+	CPodID       string       `json:"cpod_id"`       //CPOD_ID , 对于CPod的唯一标识
+	CPodVersion  string       `json:"cpod_version"`  //CPod的版本， 这里会隐含K8S的版本信息
+	GPUSummaries []GPUSummary `json:"gpu_summaries"` //GPU汇总信息，每一条代表一类GPU（如3090）
+	Nodes        []NodeInfo   `json:"nodes"`         //节点信息
+}
+
+// 做已知的GPU类型的映射，使类型名与产品前端保持一致
+func GPURename(prod string) string {
+	if prod == "NVIDIA-GeForce-RTX-3090" {
+		return "3090"
```
Comment (yzhao-2023) [inline manager/pkg/resource/resource.go:90]: 这个简化需要在 cpod 做么？

@jimgreat666 可以这样显示

File: manager/pkg/resource/resource.go [hunk 2]
```diff
@@ -67,11 +68,29 @@ type NodeInfo struct {
...
-	Nodes       []NodeInfo `json:"nodes"`        //节点信息
+	CPodID       string       `json:"cpod_id"`       //CPOD_ID , 对于CPod的唯一标识
+	CPodVersion  string       `json:"cpod_version"`  //CPod的版本， 这里会隐含K8S的版本信息
+	GPUSummaries []GPUSummary `json:"gpu_summaries"` //GPU汇总信息，每一条代表一类GPU（如3090）
+	Nodes        []NodeInfo   `json:"nodes"`         //节点信息
+}
+
+// 做已知的GPU类型的映射，使类型名与产品前端保持一致
+func GPURename(prod string) string {
+	if prod == "NVIDIA-GeForce-RTX-3090" {
```
Comment (yzhao-2023) [inline manager/pkg/resource/resource.go:89]: 可以做成 map{...}

File: manager/pkg/resource/resource.go [hunk 3]
```diff
@@ -119,6 +139,21 @@ func GetResourceInfo(CPodID string, CPodVersion string) CPodResourceInfo {
 	} else {
 		log.Println("failed retrieve node info from k8s.")
 	}
+	//stat gpus in cpod
```
Comment (yzhao-2023) [inline manager/pkg/resource/resource.go:142]: stat gpus in cpod 是什么意思？

File: manager/pkg/resource/resource.go [hunk 4]
```diff
@@ -119,6 +139,21 @@ func GetResourceInfo(CPodID string, CPodVersion string) CPodResourceInfo {
...
+	statTotal := map[[2]string]int{}
```
Comment (yzhao-2023) [inline manager/pkg/resource/resource.go:143]: [2]string
什么意思？

**Comment by yzhao-2023** [comment]
image
这个需要把 https://github.com/NascentCore/3k/pull/156 提交

**Pull request merged**